### ISSUE TITLE
How to track validation loss 

### ISSUE THREAD

**Issue by joihn** [open-mmlab/mmdetection#6236]
by using` workflow = [('train', 1),('val', 1)]`, I can track the validation AP.

How to track **both the validation AP and validation loss ?**

according to this PR https://github.com/open-mmlab/mmdetection/pull/1093 it's possible but I can't figure how

**Comment by joihn** [comment]
edit : actually the loss correctly get reported in the local logs, but does not get uploaded to "weight and bias" service.

I'm using this code in the config for weight and biases:
```
log_config = dict(
    interval=50,
    hooks=[dict(type='TextLoggerHook'),
           dict(type='WandbLoggerHook',
                init_kwargs=dict(
                    project=wnb_project_name,
                    name=f'normalized {model_name}-workers{data["workers_per_gpu"]}samples/GPU{data["samples_per_gpu"]} lr {optimizer["lr"]}',
                    entity=wnb_username)
                )]
)
```
